sdk/management/docs/DESIGN_PREVIEW.md

```diff
@@ -0,0 +1,54 @@
+# Design for Azure Management Libraries for Java (Preview)
+
+## Fluent interface
+
+### Fine control over long-running operation
+
+Resource provision takes time, a typical solution adopted by Azure services is the [long-running operation (LRO)][lro].
+
+Fluent interface does the polling operations in background, and only returns the final result.
+
+Azure Management Libraries supports fine control over the polling for certain important resources, via `Accepted` and `SyncPoller` class. Method verb is `beginCreate` and `beginDelete`.
+
+`Accepted` class provides following functionalities:
+- `ActivationResponse` via `getAcceptedResult` method provides the response of the first activation operation. Note that though it wraps a resource instance, some action on this resource instance will not work, since it is not provisioned yet.
```

Comment:
it's little confusing the method name and return type is using different term: Accepted and Activation. Is there better names? Same confusing on Class name. I can see Accepted from httpstatus code, will it be good such as beginCreateResult or beginLROResult etc?

Reply:
`ActivationResponse` extends `SimpleResponse` which implements `Response`, hence named as Response.

Activation is mostly from here https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java#L58-L61

Maybe I can change the method to `getActivationResponse`.

Reply:
Renamed to `getActivationResponse`